WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Options/JetpackRestoreOptionsViewController.swift

```diff
@@ -46,4 +56,20 @@ class JetpackRestoreOptionsViewController: BaseRestoreOptionsViewController {
         self.navigationController?.pushViewController(warningVC, animated: true)
     }
 
+    override func detailActionButtonTapped() {
+        guard let jetpackSettingsURL = URL(string: "https://wordpress.com/settings/jetpack/\(site.displayURL)") else {
+
+            let title = NSLocalizedString("Unable to visit Jetpack settings for site", comment: "Message displayed when visiting the Jetpack settings page fails.")
+            let notice = Notice(title: title)
+            ActionDispatcher.dispatch(NoticeAction.post(notice))
+
+            return
+        }
+
+        let webVC = WebViewControllerFactory.controller(url: jetpackSettingsURL)
+        let navigationVC = LightNavigationController(rootViewController: webVC)
```

Comment:
I don't think we need to force the light nav controller here. 

```suggestion
        let navigationVC = UINavigationController(rootViewController: webVC)
```

Reply:
Fixed! abf6b1a